core/src/main/java/com/graphhopper/reader/dem/AbstractEdgeElevationInterpolator.java

```diff
@@ -0,0 +1,114 @@
+package com.graphhopper.reader.dem;
+
+import com.graphhopper.coll.GHBitSet;
+import com.graphhopper.coll.GHBitSetImpl;
+import com.graphhopper.routing.util.AllEdgesIterator;
+import com.graphhopper.routing.util.DataFlagEncoder;
+import com.graphhopper.storage.GraphHopperStorage;
+import com.graphhopper.util.BreadthFirstSearch;
+import com.graphhopper.util.EdgeExplorer;
+import com.graphhopper.util.EdgeIterator;
+import com.graphhopper.util.EdgeIteratorState;
+import com.graphhopper.util.PointList;
+
+import gnu.trove.set.TIntSet;
+import gnu.trove.set.hash.TIntHashSet;
+
+public abstract class AbstractEdgeElevationInterpolator {
```

Comment:
ps. We could also use [License Maven Plugin](http://www.mojohaus.org/license-maven-plugin/), it can automatically add or check license headers. I used it for a number of GPL projects and it worked very well.